WcaOnRails/config/i18n.yml

```diff
@@ -33,6 +33,7 @@ translations:
   - file: "app/webpacker/rails_translations/:locale.json"
     patterns:
       - "*.about.*"
+      - "*.activerecord.*"
```

Comment:
Please be more conservative about which keys you're adding, The `activerecord` group contains a LOT of data (i.e. a lot of additional JSON text that we have to send to the user's browser) but I'm sure you only need a small part of it.